fcrepo-configs/src/main/resources/config/infinispan/leveldb/infinispan.xml

```diff
@@ -1,86 +1,50 @@
 <infinispan xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns="urn:infinispan:config:6.0"
-            xmlns:store="urn:infinispan:config:store:leveldb:6.0"
-            xsi:schemaLocation="urn:infinispan:config:6.0 http://www.infinispan.org/schemas/infinispan-config-6.0.xsd urn:infinispan:config:store:leveldb:6.0 http://infinispan.org/schemas/infinispan-cachestore-leveldb-config-6.0.xsd">
+            xmlns="urn:infinispan:config:7.2"
+            xmlns:store="urn:infinispan:config:store:leveldb:7.2"
+            xsi:schemaLocation="urn:infinispan:config:7.2 http://www.infinispan.org/schemas/infinispan-config-7.2.xsd urn:infinispan:config:store:leveldb:7.2 http://infinispan.org/schemas/infinispan-cachestore-leveldb-config-7.2.xsd">
+
+  <cache-container default-cache="FedoraRepository" statistics="true">
+    <jmx duplicate-domains="true"/>
+
+    <local-cache name="FedoraRepository">
+
+      <transaction transaction-manager-lookup="org.infinispan.transaction.lookup.GenericTransactionManagerLookup" mode="NON_XA" locking="PESSIMISTIC"/>
+      <eviction max-entries="500" strategy="LIRS" thread-policy="DEFAULT"/>
+      <persistence>
+        <store:leveldb-store passivation="false" path="${fcrepo.ispn.repo.cache:target/repo}/data">
```

Comment:
I tried running the modeshape clustered config (config/clustered/repository.json) and got an error unless I removed the passivation attribute from this line and line 27.